### PR TITLE
CORE-266 Default AWS tags

### PR DESCRIPTION
**Pull request by igorkotof**

### CODE REVIEW

**Review by kobrikx — APPROVED** [review]
lgtm

File: terraform/templates/backend.tf.gotmpl [hunk 1]
```diff
@@ -42,6 +42,14 @@ provider "aws" {
 provider "aws" {
   profile = var.aws_profile
   region  = var.aws_region
+  default_tags {
+    tags = {
+      terraform          = "true"
+      namespace          = "{{ .Env.NAMESPACE }}"
+      environment        = "{{ .Env.ENV }}"
```
Comment (AutomationD) [inline terraform/templates/backend.tf.gotmpl:49]: Should this be `env` rather?

**Review by AutomationD — CHANGES_REQUESTED** [review]
Great solution! 

Question about name of the `env`

File: terraform/templates/backend.tf.gotmpl (hunk 1 above)
Comment (igorkotof) [reply terraform/templates/backend.tf.gotmpl:49]: it's up to you :) let me rename

**Review by AutomationD — APPROVED** [review]
Looks good.

**Pull request merged**